docs/src/transforms.md

```diff
@@ -39,10 +39,10 @@ Future transforms won't be added to the public API while in alpha. But in these
 | [Protect](#protect)                                      | ❌          | Alpha                 |
 | [QueryCounter](#querycounter)                            | ❌          | Alpha                 |
 | [QueryTypeFilter](#querytypefilter)                      | ❌          | Alpha                 |
-| [RedisCache](#rediscache)                                | ❌          | Alpha                 |
-| [RedisClusterPortsRewrite](#redisclusterportsrewrite)    | ❌          | Beta                  |
-| [RedisSinkCluster](#redissinkcluster)                    | ✅          | Beta                  |
-| [RedisSinkSingle](#redissinksingle)                      | ✅          | Beta                  |
+| [ValkeyCache](#rediscache)                               | ❌          | Alpha                 |
+| [ValkeyClusterPortsRewrite](#redisclusterportsrewrite)   | ❌          | Beta                  |
+| [ValkeySinkCluster](#redissinkcluster)                   | ✅          | Beta                  |
+| [ValkeySinkSingle](#redissinksingle)                     | ✅          | Beta                  |
```

Comment:
I think the links to headings need to be updated as well

Reply:
good catch, seems I only replaced upper case `Redis` not `redis`, thanks!